src/lib/jest_runner.ts

```diff
@@ -23,6 +23,10 @@ export class JestRunner extends EventEmitter {
     start = () => {
         const tempJSON = tmpdir() + "/vscode-jest_runner.json";
         const args = ['--json', '--useStderr', '--watch', "--jsonOutputFile", tempJSON];
+        if (this.workspace.pathToConfig !== "") {
+            args.push("--config");
+            args.push(this.workspace.pathToConfig);
+        }
```

Comment:
you want this inside the jest_process instead, otherwise it won't happen on the other times we run Jest (generating configs, updating screenshots etc )